dev/com.ibm.ws.wssecurity_fat.wsscxf.1/fat/src/com/ibm/ws/wssecurity/fat/cxf/x509token/CxfX509OverRideTests.java

```diff
@@ -129,7 +129,7 @@ public static void setUp() throws Exception {
     //
 
     @Test
-    @AllowedFFDC(value = { "org.apache.wss4j.common.ext.WSSecurityException", "java.net.MalformedURLException" }, repeatAction = { EE8FeatureReplacementAction.ID })
+    @AllowedFFDC(value = { "java.net.MalformedURLException" }, repeatAction = { EE8FeatureReplacementAction.ID })
```

Comment:
The exception `"org.apache.wss4j.common.ext.WSSecurityException"` is missed? or it's not needed? This exception is usually used for EE8.

Reply:
ditto

Reply:
ok thanks for clarification.